walnut/cmd/convert.go

```diff
@@ -57,30 +67,63 @@ specify which modules should be used when generating task templates. Control-OCC
 			dplDump, err := converter.DPLImporter(file)
 			taskClass, err := converter.ExtractTaskClasses(dplDump, modules)
 
-			err = converter.GenerateTaskTemplate(taskClass, nameOfDump)
+			if outputDir == "" {
+				outputDir, _ = os.Getwd()
+			}
+			outputDir, _ = homedir.Expand(outputDir)
+
+			runGitCmd := func(args []string) string {
+				cmd := exec.Command("git", args...)
+				cmd.Dir = outputDir
+				out, err := cmd.Output()
+				if err != nil {
+					log.Fatal(err)
+				}
+
+				return string(out)
+			}
+
+			isGitRepo, _ := strconv.ParseBool(strings.TrimSpace(
+				runGitCmd([]string{"rev-parse", "--is-inside-work-tree"})))
+
+			err = converter.GenerateTaskTemplate(taskClass, outputDir)
 			if err != nil {
 				err = fmt.Errorf("conversion to task failed for %s: %w", dumpFile, err)
 				fmt.Println(err.Error())
 				os.Exit(1)
 			}
 
 			role, err := workflow.LoadDPL(taskClass, nameOfDump)
-			err = converter.GenerateWorkflowTemplate(role, nameOfDump)
+			err = converter.GenerateWorkflowTemplate(role, outputDir)
 			if err != nil {
 				err = fmt.Errorf("conversion to workflow failed for %s: %w", dumpFile, err)
 				fmt.Println(err.Error())
 				os.Exit(1)
 			}
+
+			if isGitRepo {
+				fmt.Print("Press 'Enter' to view git diff...")
+				bufio.NewReader(os.Stdin).ReadBytes('\n')
```

Comment:
Can this be made nicer with `promptui` instead of reading an `Enter`? Something like `Would you like to view the git diff? [Y/n]` so pressing `Enter` still yields the same result, but with the option to refuse.